pegasus/sites.v3/code.org/homepage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "pages": {
+    "homepage": "higherPower"
+  },
+
+  "banners": {
+    "higherPower": {
+      "environments": ["development", "staging", "test"],
+      "desktopImage": "/images/marketing/homepage-higher-power-background.png",
+      "items": [
+        {
+          "type": "image",
+          "url": "/images/marketing/homepage-higher-power-logo.png"
+        },
+        {
+          "type": "text",
+          "text": "homepage_higher_power_body"
```

Comment:
Does marketing have the ability/ know the process around adding text to the i18n files?

Reply:
That's a great question, and I hope the answer is yes.  It's one of our oldest gsheets for content editors, and certainly in the past we expected content editors to be able to work with it.  I'll be available to help anybody who needs it.